src/index.html

```diff
@@ -18,5 +23,87 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+
+    <header class="header">
+      <a
+        class="logo"
+        href="https://commons.wikimedia.org/wiki/File:LOGO_moyo_1.png"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class `nav-list` should be `nav__list` to maintain BEM naming conventions and represent that it is an element of the `nav` block.